.github/workflows/publish-chromatic.yml

```diff
@@ -0,0 +1,48 @@
+name: 'Publish Chromatic'
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_dispatch:
+
+jobs:
+  chromatic_deployment:
+    name: Publish Chromatic
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
This could use

    with:
        fetch-depth: 0

to use a shallow checkout.

Reply:
Looks like the chromatic github action asks for this too:
![image](https://user-images.githubusercontent.com/2684369/122135473-6cbded00-cdf5-11eb-8a77-ea5661532b57.png)
https://github.com/chromaui/action